source/_components/device_tracker.icloud.markdown

```diff
@@ -33,9 +33,11 @@ Configuration variables:
 - **username** (*Required*): The username for the iCloud account.
 - **password** (*Required*): The password for your given username.
 - **account_name** (*Optional*): The friendly name for the account_name. If this isn't given, it will use the account_name of the username (so the part before the `@` in the email address).
+- **max_interval** (*Optional*): Maximum interval in minutes between subsequent location checks. This tracker uses dynamic intervals for requesting location updates. When iphone is stationary, interval will eventually be set to `max_interval`. This saves battery. E.g. setting it to 10 means that when you start moving after being stationary, maximum 10 min will pass until your location starts updating. Default is 30 min. Very minimum value is 1 min.
```

Comment:
Can you simplify what does `For example, setting it to 10 means that when you start moving after being stationary, maximum 10 min will pass until your location starts updating.` mean? I am guessing, if the phone is stationary, it updates every 10 minutes. What about when the phone is not stationary?

Reply:
Rephrased a bit. Let me know if it still sounds confusing.